source/MainWindow.cpp

```diff
@@ -147,6 +145,7 @@ MainWindow::MessageReceived(BMessage* message)
 		}
 		default:
 			BWindow::MessageReceived(message);
+			message->PrintToStream();
```

Comment:
I suppose one could leave a bit of debugging in the source, but it should be commented out when not needed.

Reply:
your right!